src/material-experimental/mdc-tabs/testing/tab-link-harness.ts

```diff
@@ -33,6 +33,12 @@ export class MatTabLinkHarness extends ComponentHarness {
     return (await this.host()).text();
   }
 
+  /** Gets the value of the `aria-controls` attribute. */
+  async getAriaControls(): Promise<string | null> {
```

Comment:
I don't think that we should be exposing this as a public API. You should be able to do `link.host().getAttribute()` in the nav bar harness.

Reply:
Done.